src/ios/DateTimePicker.m

```diff
@@ -123,6 +123,8 @@ - (void)configureDatePicker:(NSMutableDictionary *)optionsOrNil datePicker:(UIDa
 {
     NSString *mode = [optionsOrNil objectForKey:@"mode"];
     long long ticks = [[optionsOrNil objectForKey:@"ticks"] longLongValue];
+    long long minDate = [[optionsOrNil objectForKey:@"minDate"] longLongValue];
```

Comment:
If minDate or maxDate aren't specified in options, this crashes, as [optionsOrNil objectForKey:@"minDate"] results in Null, which longLongValue can't be called on.

Reply:
Should be fixed now, see #22